src/JuliusSweetland.OptiKey/UI/Views/Management/PointingAndSelectingView.xaml

```diff
@@ -184,6 +184,35 @@
                     <controls:NumericUpDown Grid.Row="4" Grid.Column="1" TextAlignment="Left"
                                             Minimum="1" Interval="10" StringFormat="###,##0"
                                             Value="{Binding PointTtlInMs, Mode=TwoWay}" />
+
+                    <TextBlock Grid.Row="4" Grid.Column="0" Text="{x:Static resx:Resources.HIDE_CURSOR_LABEL}" 
```

Comment:
Is this not rendering over the TIME_UNTIL_POINT_BECOMES_IRRELEVENT_LABEL setting above, i.e. both are in Row=4 ?